cinema/views.py

```diff
@@ -0,0 +1,41 @@
+from django.shortcuts import get_object_or_404
+from rest_framework import status
+from rest_framework.decorators import api_view
+from rest_framework.response import Response
+
+from cinema.models import Movie
+from cinema.serializers import MovieSerializer
+
+
+@api_view(["GET", "POST"])
+def movie_list(request) -> Response:
+    if request.method == "GET":
+        movies = Movie.objects.all()
+        serializer = MovieSerializer(movies, many=True)
+        return Response(serializer.data, status=status.HTTP_200_OK)
+
+    if request.method == "POST":
+        serializer = MovieSerializer(data=request.data)
+        if serializer.is_valid():

```

Comment:
```suggestion
        serializer.is_valid(raise_exception=True):

```
In this way the 400 response will be returned automatically when the data is not valid, and you don't need to return it by yourself

Reply:
Fix everywhere